app/assets/javascripts/expand.js

```diff
@@ -31,22 +31,22 @@ $(document).ready(function() {
       var para5 = document.createElement("p");
 
       var desc_node = document.createTextNode(description);
+      var complete_node = null;
 
       if (completion == true) {
-        var complete_node = document.createTextNode("Completed");
+        complete_node = document.createTextNode("Completed");
```

Comment:
Identifier 'complete_node' is not in camel case.<br>Mixed double and single quotes.